### DISCUSSION TITLE
キーボール６１のキーソケットのランド剥がれ

### DISCUSSION THREAD

**Discussion by TakashiTatsumi** [Yowkees/keyball#627] [Q&A]
お世話になります。
画像の赤枠の部分のランドが剥離してしまいました。
回路図が読めずジャンパの位置がわからないのですがご教示いただけると助かります。

![無題](https://github.com/user-attachments/assets/85d14536-7aad-4fd7-9ae1-e3b705927135)

**Comment by TakashiTatsumi** [comment]
![image](https://github.com/user-attachments/assets/755b6424-1afa-48cd-bd81-c3b97de66136)
下のダイオードとつなげると意図した動きになった。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
ご自身で解決していただきありがとうございます。ご指摘の２点で間違いありません。

ご対応ありがとうございました。ぜひ末永くご愛用いただければ幸いです。
